src/node/rpc/test/member_voting_test.cpp

```diff
@@ -1053,6 +1073,11 @@ DOCTEST_TEST_CASE("Propose raw writes")
       const bool should_succeed = pro_votes > n_members / 2;
       NetworkState network;
       network.tables->set_encryptor(encryptor);
+      auto history =
+        std::make_shared<ccf::NullTxHistory>(*network.tables, 0, *kp);
+      network.tables->set_history(history);
+      auto consensus = std::make_shared<aft::StubCFTConsensus>();
+      network.tables->set_consensus(consensus);
       auto gen_tx = network.tables->create_tx();
       GenesisGenerator gen(network, gen_tx);
       gen.init_values();
```

Comment:
Can we bury this (and `gen.create_service()`, and maybe some other lines that are _often_ called + harmless where they're currently not called) in a `init_network` or similar?

Reply:
Sure